pom.xml

```diff
@@ -526,5 +526,6 @@
             **/*Component.java,
             **/*Holder.java
         </sonar.coverage.exclusions>
+        <jsr311.api.version>1.1.1</jsr311.api.version>
```

Comment:
Shall we move this above the sonar coverage rules so that all the versions are listed in a single block?

Reply:
Resolved by 82dbf0c1e232702704d16820175afc16e45a4ec2